turbo.json

```diff
@@ -17,7 +17,9 @@
       "inputs": ["manifests"],
       "outputs": ["dist/**"]
     },
-    "lint:ci": {},
+    "lint:ci": {
+      "dependsOn": ["^build"]
```

Comment:
why does lint depend on build outputs?

Reply:
you're right. I just read in the docs that empty configs are bad but I agree that in this case it's fine